lib/urlwatch/jobs.py

```diff
@@ -280,6 +286,25 @@ def add_custom_headers(self, headers):
             headers.pop(header, None)
         headers.update(self.headers)
 
+    def format_error(self, exception, tb):
+        if isinstance(exception, requests.exceptions.RequestException):
+            # Instead of a full traceback, just show the HTTP error
+            return str(exception)
+        return tb
+
+    def ignore_error(self, exception):
+        if isinstance(exception, requests.exceptions.ConnectionError) and self.ignore_connection_errors:
+            return True
+        elif isinstance(exception, requests.exceptions.HTTPError):
+            status_code = exception.response.status_code
+            if isinstance(self.ignore_http_error_codes, int) and self.ignore_http_error_codes == status_code:
+                return True
+            elif isinstance(self.ignore_http_error_codes, str):
+                ignored_codes = [s.strip().lower() for s in self.ignore_http_error_codes.split(',')]
+                if str(status_code) in ignored_codes or '%sxx' % (status_code // 100) in ignored_codes:
```

Comment:
This is smart, but should we also support `ignore_http_error_codes` being a list of strings/numbers in the YAML? And what if the user explicitly (erroneously) specifies a HTTP status code as a string? (e.g. `"404"`) Should that be handled too?